<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wifi_status">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp1_label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>345</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="hum1_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>345</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp2_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>345</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
    <widget class="QLabel" name="hum2_label">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>345</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
    <widget class="QLabel" name="living_window_label">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>375</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
    <widget class="QLabel" name="living_light_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>373</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
    <widget class="QLabel" name="bathroom_fan_label">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>375</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
    <widget class="QLabel" name="door_label">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>375</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
    <widget class="QLabel" name="autodoor_label">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>375</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="kitchen_hood_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>625</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="kitchen_smoke_label">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>630</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostroom_curtain_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="hostroom_light_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>

    <widget class="QLabel" name="music_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>

   <widget class="QLabel" name="music_name_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>


   <widget class="QLabel" name="secound_curtain_label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="secound_light_label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="study_desk_label">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="study_light_label">
    <property name="geometry">
     <rect>
      <x>1185</x>
      <y>630</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>


  </widget>
  
 </widget>
 <resources/>
 <connections/>
</ui>
